packages/slate/test/interfaces/Editor/unhangRange/block-hanging.tsx

```diff
@@ -5,8 +5,9 @@ import { jsx } from '../../..'
 export const input = (
   <editor>
     <block>
+      wo
```

Comment:
Seems safer to keep the old test and add a new test? (so that there's a test for the start point at the start or in the middle?